src/helpers/normalize.js

```diff
@@ -225,10 +228,12 @@ const processContentEntry = async ({ schema, locale, entry, gatsbyHelpers }) =>
       fieldType,
       editorContent,
       nodeId,
+      index,
       gatsbyHelpers
     })
 
     childrenNodes.push(contentNode.id)
+    index = index + 1
```

Comment:
Changing this to `index += 1` should resolve the linting error.

Reply:
I see that this issue is fixed in the develop branch. This merge request can be rejected.